src/shell.ts

```diff
@@ -187,7 +187,7 @@ export function shellEnvironment(baseEnvironment: any): any {
         }
     }
 
-    const kubeconfig = getActiveKubeconfig();
+    const kubeconfig = getKubeconfig();
     if (kubeconfig) {
```

Comment:
>if (kubeconfig) { [](start = 4, length = 17)

The behavior changed a bit here compared to the previous version of the code, as now this condition will always be true (before we were only looking at the value of vscode.workspace.getConfiguration(EXTENSION_CONFIG_KEY)[KUBECONFIG_PATH_KEY]).
@itowlson > Please let us know if you expect this to create any side-effects.

Reply:
I think this is pretty safe to drop the check.  It may result in a superfluous overwrite of the KUBECONFIG env var but it should be harmless.

Reply:
Of course one of our users will surprise me with some edge case where it does cause a change because that is how kubectl rolls.  _mutter mutter_

Reply:
I got rid of the check, but please let me know if you think of any such edge case!